pages/interview-process.md

```diff
@@ -3,6 +3,10 @@ permalink: /interview-process/
 title: Interview process
 ---
 
+### Application review
+
+The first step of the process will be a review of the information that you have submitted as part of your application. If your application is found to be qualified, you will proceed to the interview process.
```

Comment:
@rooneywp Worth saying something here about the term 'qualified' or considering using a different word? I ask because it both is loaded and has a technically specific meaning.